inc/saiport.h

```diff
@@ -47,6 +47,19 @@ typedef enum _sai_port_type_t
 } sai_port_type_t;
 
 /**
+* @brief Attribute data for #SAI_PORT_ATTR_BIND_MODE
+*/
+typedef enum _sai_port_bind_mode_t
+{
+    /** Port */
+    SAI_PORT_BIND_MODE_PORT,
+
+    /** Sub port */
+    SAI_PORT_BIND_MODE_SUB_PORT,
+
+} sai_port_bind_mode_t;
```

Comment:
Section 2 "sub port interface" explains about sub port and legacy port modes

Section 6.1 gives examples of using sub port
//configure port 3,4,5,6 to sub port mode  
sai_attribute_t port_attr;
port_attr.id=SAI_PORT_BIND_MODE;
port_attr.value= SAI_PORT_BIND_MODE_SUB_PORT;
sai_set_port_attribute_fn(port_3,port_attr);
sai_set_port_attribute_fn(port_4,port_attr);

Section 6.2 gives example of using port mode
sai_attribute_t port_attr;
port_attr.id=SAI_PORT_BIND_MODE;
port_attr.value= SAI_PORT_BIND_MODE_PORT;
sai_set_port_attribute_fn(port_3,port_attr);
sai_set_port_attribute_fn(port_4,port_attr);